Assets/Scripts/TCPToolkit/TCPToolkit.cs

```diff
@@ -63,9 +63,15 @@ public static Packet FirstPacketFromBytes(byte[] bytes)
                     }
 
                     int byteCount = packet.DataSize + TCP_HEADER_SIZE;
+                    
+                    if (bytes.Length < byteCount)
+                    {
+                        Debug.Log("Packet not ready to be built yet: " + bytes.Length + " vs " + byteCount);
```

Comment:
Same as the other, can we change to: Debug.Log(String.Format("Packet not ready to be built yet: {0} vs {1}", bytes.Length, byteCount)); ?

Reply:
Apparently, I'm very wrong in c# as seen [here](https://stackoverflow.com/questions/925423/is-it-better-practice-to-use-string-format-over-string-concatenation-in-java). Therefore, I'll shut up and approve this PR.